src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,13 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService validationUser = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        boolean result = false;
+        if (validationUser.findByEmail(email) != null) {
+            result = validationUser.findByEmail(email).getPassword().equals(password);
```

Comment:
you can save found User in a local variable
If any questions - ask in Java chat, do not use PR comments for communication)
![image](https://github.com/mate-academy/jv-oop-lesson-0/assets/20326858/cfda62a4-b539-42af-bfa0-511b7743c801)